Base/usr/share/man/man7/KeyboardShortcuts.md

```diff
@@ -0,0 +1,65 @@
+![Keyboard Icon](/res/icons/32x32/app-keyboard-settings.png)
+
+# Keyboard Shortcuts
+
+This is a cheatsheet of common keyboard shortcuts in SerenityOS
+
+Related: [Tips and Tricks](help://man/7/Tips-and-Tricks)
+
+### Keywords
+* `Super` means the ⊞ Windows or ⌘ Command key
+* `Alt` is equivalent to ⌥ Option on a Mac
+  
+## General
+| Shortcut      | Action                                                    |
+|---------------|-----------------------------------------------------------|
+| `Super`       | Open the System Menu (navigate via arrow keys)            |
+| `Super+Space` | Summon Assistant (system search and application launcher) |
+
+## Window Management
+| Shortcut                 | Action                                                                |
+|--------------------------|-----------------------------------------------------------------------|
+| `Super+Up`               | Maximize the window                                                   |
+| `Super+Down`             | Minimize the window (or un-maximize)                                  |
+| `Super+Left`             | Snap window to left half of the screen                                |
+| `Super+Right`            | Snap window to right half of the screen                               |
+| `Super+LeftMouseButton`  | Move window (grabbing anywhere within)                                |
+| `Super+RightMouseButton` | Resize window (in the direction of mouse drag)                        |
+| `F11`                    | Fullscreen mode (not available in all applications)                   |
```

Comment:
not available in all is an understatement, it's more like "only available in some"